test/functionalTest/cases/647_crash_with_compounds/PUT_v1_contextEntities_E1_attributes_A1.test

```diff
@@ -102,20 +102,27 @@ payload='<?xml version="1.0" encoding="UTF-8"?>
           <name>A</name>
           <type>AT1</type>
           <contextValue>3</contextValue>
+          <metadata>
+            <contextMetadata>
+              <name>ID</name>
+              <type>none</type>
+              <value>253</value>
+            </contextMetadata>
+          </metadata>
```

Comment:
Yes, it seems to be a bug related with metadata omission in the conv op.

We should include an entry in CHANGES_NEXT_RELEASE about this happy fix :)